node/src/service.rs

```diff
@@ -10,9 +11,10 @@ use cumulus_client_service::{
     StartFullNodeParams,
 };
 use datahighway_runtime::{
```

Comment:
please order the imports similar to how they did it in substrate-parachain-template https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/node/src/service.rs#L1

i.e. under comments `// std`, `// Local Runtime Types`, `// Substrate Imports`, `// Runtime type overrides`

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231